README.md

```diff
@@ -55,6 +55,10 @@ The generation task currently creates the following signatures:
 - Named scopes
 - Model relation class
 
+Note that we do not generate sigs for [overridden methods](https://github.com/chanzuckerberg/sorbet-rails/issues/19). It would show this message in the log when running the rake task:
```

Comment:
```suggestion
Note that we do not generate signatures for [overridden methods](https://github.com/chanzuckerberg/sorbet-rails/issues/19). It would show this message in the log when running the Rake task:
```